### ISSUE TITLE
Epsilon implicitly set to 1

### ISSUE THREAD

**Issue by mlpanda** [gabrielhuang/reptile-pytorch#3]
Hi Gabriel, 

Thanks a lot for the nice code! 

When you add the parameter difference in `ReptileModel.point_grad_to`, you add the difference directly rather than multiplying with epsilon as in the paper:

`p.grad.data.add_(p.data - target_p.data)`

Is there any specific reason for this? 

Thanks,
Marco

**Comment by gabrielhuang** [comment]
Hi Marco,

You're welcome !

There is no need to multiply the difference,
the meta-learning rate is handled by the meta-optimizer.

For instance in the omniglot case,

the actual gradient step is taken at this line:
https://github.com/gabrielhuang/reptile-pytorch/blob/4c85a5da189498994b414de7fe7a401125345a62/train_omniglot.py#L228

and the meta-optimizer learning rate is set at this line
https://github.com/gabrielhuang/reptile-pytorch/blob/4c85a5da189498994b414de7fe7a401125345a62/train_omniglot.py#L176

**Comment by mlpanda** [comment]
Thanks for the prompt reply, it makes sense - at least given that SGD is used as optimizer :-)

**Comment by gabrielhuang** [comment]
No problem ! You can also explore different combinations of optimizers for the inner and outer loop (SGD, Adam, etc ...)
